main.go

```diff
@@ -21,18 +22,20 @@ const (
 // functions in the bd-service proto file
 type Router struct {
 	client bpb.BirthdayFunctionsClient
+	bpb.UnimplementedBirthdayFunctionsServer
 }
 
 // Func that creates a new client and connects
 // to the bd-service client through the proto file
 func initClientConnection() bpb.BirthdayFunctionsClient {
 
-	envError := godotenv.Load()
-	if envError != nil {
-		log.Fatal("Error loading .env file! error: ", envError)
+	// Loading the dotenv parameters
+	config, err := util.LoadConfig(".")
+	if err != nil {
+		log.Fatal("cannot load config:", err)
 	}
 
-	address := os.Getenv("ADDRESS") + os.Getenv("PORT")
+	address := config.Address + config.Port
```

Comment:
Use one env for the address and port. See our api-gateway

Reply:
done